subsys/bluetooth/controller/ll_sw/ull_connections.c

```diff
@@ -907,8 +947,89 @@ static inline void disable(uint16_t handle)
 
 static void conn_cleanup(struct ll_conn *conn, uint8_t reason)
 {
+	struct lll_conn *lll = &conn->lll;
+	struct node_rx_pdu *rx;
+	uint32_t ticker_status;
+
+	/* Only termination structure is populated here in ULL context
+	 * but the actual enqueue happens in the LLL context in
+	 * tx_lll_flush. The reason being to avoid passing the reason
+	 * value and handle through the mayfly scheduling of the
+	 * tx_lll_flush.
+	 */
+	rx = (void *)&conn->terminate.node_rx;
+	rx->hdr.handle = conn->lll.handle;
+	rx->hdr.type = NODE_RX_TYPE_TERMINATE;
+	*((uint8_t *)rx->pdu) = reason;
+
+	/* flush demux-ed Tx buffer still in ULL context */
+	tx_ull_flush(conn);
+
+	/* Stop Master or Slave role ticker */
+	ticker_status = ticker_stop(TICKER_INSTANCE_ID_CTLR,
+				    TICKER_USER_ID_ULL_HIGH,
+				    TICKER_ID_CONN_BASE + lll->handle,
+				    ticker_op_stop_cb, (void *)lll);
+	LL_ASSERT((ticker_status == TICKER_STATUS_SUCCESS) ||
+		  (ticker_status == TICKER_STATUS_BUSY));
+
+	/* Invalidate the connection context */
+	lll->handle = 0xFFFF;
+
+	/* Demux and flush Tx PDUs that remain enqueued in thread context */
+	ull_conn_tx_demux(UINT8_MAX);
 }
 
+static void tx_ull_flush(struct ll_conn *conn)
+{
```

Comment:
Empty on purpose?

Reply:
Yes, It's updated in a later commit (https://github.com/zephyrproject-rtos/zephyr/pull/32332/commits/82abbb3714d7f9b51101616581de13886681c77c)

Reply:
I see that now - sorry :-)